airflow/models/expandinput.py

```diff
@@ -217,6 +217,10 @@ class ListOfDictsExpandInput(NamedTuple):
 
     value: OperatorExpandKwargsArgument
 
+    def _iter_parse_time_resolved_kwargs(self, mapping) -> Iterable[tuple[str, Sized]]:
+        """Generate kwargs with values available on parse-time."""
+        return ((k, v) for k, v in mapping.items() if _is_parse_time_mappable(v))
```

Comment:
Do we really need this function that’s only used in one place? Especially this one has a different signature from the function on `DictOfListsExpandInput` with the same name.

Reply:
Actually we don't